cloudera-integration/csd/descriptor/service.sdl

```diff
@@ -103,6 +103,81 @@
         ]
       }
     },
+    {
+      "name": "LOG_COLLECTOR_KAFKA",
+      "label": "Log Collector Kafka",
+      "pluralLabel": "Log Collectors Kafka",
+      "inExpressWizard": true,
```

Comment:
I don't think this line does anything. Can you check if `inExpressWizard` works for anything other than individual parameters?

Reply:
removed this for Collection Roller, Alert Engine, Log Collector, Log Collector Kafka